plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -126,23 +124,43 @@ class GeneralServiceFormSection extends Component {
     }
 
     return (
-      <p>
-        {'Need to run a service with multiple containers? '}
-        <a className="clickable" onClick={this.handleOpenConvertToPodModal}>
-          Add another container
-        </a>.
-      </p>
+      <div className="pod pod-narrow pod-short">
```

Comment:
It seems odd that "Need to run a service..." isn't left aligned.

![image](https://cloud.githubusercontent.com/assets/15963/22480285/b463c6b0-e7a5-11e6-8e7d-fddf0a57de29.png)